### ISSUE TITLE
CoC-Specific Settings Not Reflected in Downloads

### ISSUE THREAD

**Issue by vlopez0603** [abtassociates/eva#86]
**Describe the bug**
The number of Possible Missed Exits in exports and Summary tables should be impacted depending on the CoC-Specific Settings, but the number of Possible Missed Exits does not change.

**To Reproduce**
1. Upload 11_15 dataset
2. Click on Client Counts
3. Select SO - ESG CV project
4. Search for "Current" in table to see that there are 3 enrollments that are over 90 days (the default CoC setting for Street Outreach)
5. Go to System-level tab
6. Download export
7. Filter Summary tab of export for "Missed Exits"
8. SO - ESG CV project is not included in list of projects with Missed Exits issue
9. Go to Organization-level tab
10. Download export
11. Filter Summary tab of export for "Missed Exits"
12. SO - ESG CV project is not included in list of projects with Missed Exits issue
13. Refresh app
14. Update all (or just Street Outreach) CoC specific settings to 5
15. Repeat steps 1-11 to verify that SO - ESG CV project is still not included in list of projects with Missed Exits issue
16. Go to Home tab
17. Update CoC-specific settings without refreshing the app
18. Repeat steps 1-11 to verify that SO - ESG CV project is still not included in list of projects with Missed Exits issue

**Expected behavior**
When CoC specific settings are updated for specific project types, the number of Possible Missed Exits should be impacted--both in the Summary Table on the Organization-level tab and in the system level and org level exports.

**Additional context**
![c7f4c0ed-49c9-44c1-9dcb-28cd98166421](https://user-images.githubusercontent.com/96145208/206298933-3a192e71-ff16-4c88-9f05-f8a6af6e5b10.jpg)

![7a96f583-ff2d-46b6-b169-e71639f87220](https://user-images.githubusercontent.com/96145208/206298914-7b85e7a6-05ad-4220-9547-1e8c97054731.jpg)

**Comment by kiadso** [comment]
Great catch, thank you! Fixed in PR #92

**Comment by kiadso** [comment]
Fixed on test and live. Link to the live site will be released next week! Thank you again for your testing and the issues you entered!

**Comment by kiadso** [comment]
Thanks for raising this again Victoria!

**Comment by kiadso** [comment]
Hi @vlopez0603 In testing this, I think what happened is that we renamed the Issue for the ones being calculated from the CoC-specific settings to "Days Enrollment Active Exceeds CoC-specific Settings" so searching for "Missed Exit" will not yield any results. I'm seeing that SO - ESG CV has 3 results when the settings are at 90 and if I change it to 204, it switches to 1 result. I searched on "coc" instead of "missed ex".
